x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/metadata/table.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiText, EuiFlexGroup, EuiFlexItem, EuiLink, EuiBasicTable } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useCallback, useMemo, useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+interface Row {
+  name: string;
+  value: string | string[] | undefined;
+}
+
+interface Props {
+  rows: Row[];
+}
+
+/**
+ * Columns translations
+ */
+const FIELD_LABEL = i18n.translate('xpack.infra.hostsViewPage.hostDetail.metadata.field', {
+  defaultMessage: 'Field',
+});
+
+const VALUE_LABEL = i18n.translate('xpack.infra.hostsViewPage.hostDetail.metadata.value', {
+  defaultMessage: 'Value',
+});
+
+export const Table = (props: Props) => {
+  const { rows } = props;
+  const columns = useMemo(
+    () => [
+      {
+        field: 'name',
+        name: FIELD_LABEL,
+        width: '35%',
+        sortable: false,
+        render: (name: string) => <EuiText size="s">{name}</EuiText>,
+      },
+      {
+        field: 'value',
+        name: VALUE_LABEL,
+        width: '65%',
+        sortable: false,
+        render: (_name: string, item: Row) => {
+          return (
+            <>
+              {!Array.isArray(item.value) && item.value}
+              {Array.isArray(item.value) && <ExpandableContent values={item.value} />}
+            </>
+          );
+        },
```

Comment:
**nit:** `ExpandableContent` now convert the passed values always into an array, so this distinction is not required anymore:
```js
render: (_name: string, item: Row) => <ExpandableContent values={item.value} />
```

Reply:
Good catch, changed here and I will update the other PR as well